futurenhs.api/FutureNHS.Api/Controllers/ContentController.cs

```diff
@@ -36,15 +37,15 @@ public async Task<IActionResult> GetImageAsync(Guid imageId)
         [HttpPost]
         [Route("page/{userId:guid}/{groupId}")]
         [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(Guid))]
-        public async Task<IActionResult> CreatePageAsync(Guid userId, Guid groupId, [FromBody] CreatePageRequest createRequest, CancellationToken cancellationToken)
+        public async Task<IActionResult> CreatePageAsync(Guid userId, Guid groupId, [FromBody] GeneralWebPageCreateRequest createRequest, CancellationToken cancellationToken)
```

Comment:
A few unnecessary async state machines in here.

I take it we're going to validate that the user has permission to perform these actions at a later date? currently we're not using `userId` and not validating that that user has permissions to perform these actions

Reply:
Thanks Matt, yes will add these at a later date